docs/system-requirements.md

```diff
@@ -45,8 +45,25 @@ The specific storage consumption for k0s is as follows:
 
 ## Host operating system
 
-- Linux (see [Linux specific requirements] for details)
-- Windows Server 2019
+k0s is tested and known to work on the following operating systems and versions:
+
+| Operating System              | Version/Notes                                                                           |
```

Comment:
I'd probably add some comments both here and in `ostests-nightly.yaml` to note that when one place is updated, the other should be checked as well.